packages/client/src/checkout/types/postCheckoutOrder.types.ts

```diff
@@ -9,6 +9,9 @@ export type PostCheckoutOrderData = {
   guestUserEmail?: string;
   shippingMode?: ShippingMode;
   metadata?: PostCheckoutOrderMetadata;
+  tenantId?: number;
+  customerId?: number;
+  isGuestUser?: boolean;
```

Comment:
Question: Does the server validate if invalid combination of parameters are sent? For example, I could send `guestUserEmail` with a value but set `isGuestUser` to false. What will happen in that case? If it is invalid, then I think this type should be divided in 2: One when `guestUserEmail` is a string which will restrict the type of `isGuestUser` to `true | undefined` and another where `guestUserEmail` is undefined and `isGuestUser` can be then `boolean | undefined`.